### ISSUE TITLE
graphql-codegen recipe does not work

### ISSUE THREAD

**Issue by rickyhopkins** [Urigo/graphql-modules#811]
I have been trying to get a base app up and running using apollo server and graphql-modules.

I have gone through the [development environment setup steps](https://graphql-modules.com/docs/recipes/development-environment) using the ts-node method. This all works fine and I create a super simple Module from the examples:

```typescript
import { GraphQLModule } from '@graphql-modules/core';
import gql from 'graphql-tag';

export const AppModule = new GraphQLModule({
    typeDefs: gql`
        type Query {
            myData: Data
        }

        type Data {
            field: String
        }
    `
});
```

But when I try to follow the [graphql-codegen steps](https://graphql-modules.com/docs/recipes/graphql-code-generator) to generate the typings I get the error
`bad indentation of a sequence entry in "D:\projects\bootstrap\server\codegen.yml" at line 12, column 11:  - @graphql-modules/core#ModuleContext
`

The indentation seems fine and when I change that line to `contextType: '@graphql-modules/core#ModuleContext'` it passes that step but I then get an error saying that it cannot find my schema file.

`Failed to load schema from ./src/schema.ts: Unable to load from file "D:\projects\bootstrap\server\src\schema.ts": Cannot find module 'D:projectootstrapserversrcschema.ts'`

No idea where to go from here. But I have tried starting again a few times with a blank slate each time thinking that I have messed something up in the setup instructions but I keep coming across the same issue.

**Comment by IsenrichO** [comment]
I can confirm that I'm seeing the same issue.

**Comment by wuichen** [comment]
im seeing the same issue as well

**Comment by dotansimha** [comment]
@kamilkisiela is now working on a built-in integration of GraphQL Modules and GraphQL Codegen. You can find the progress here: https://github.com/dotansimha/graphql-code-generator/pull/3923

**Comment by kamilkisiela** [comment]
We released a new major version (v1.0) of GraphQL Modules. Here's the [migration guide](https://graphql-modules.com/docs/next/recipes/migration).

The new version solves the issue, so I'm closing the thread.

Please try to upgrade. If you can't and still want to use the legacy version, please open a new issue.
